### ISSUE TITLE
Error, Notification: Label(Filter) is not associated with the 'All' combo box control present under 'Console' blade.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1053]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1245675

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section.
Create 'Sample Database'. 
Navigate to Expand/Collapsed button in the bottom and select it.
Navigate through all the controls like 'Filter' combo box.
Press F12 key to open the Developers Tool and inspect the code.
Verify the issue.

Actual Result:
Label(Filter) is not associated with the 'All' combo box control present under 'Console' blade due to which the screen reader is not reading the label information.

Expected Result:
Label(Filter) should be associated with the 'All' combo box control present under 'Console' blade so that  screen reader will read the label information along with combo box information.

![image](https://user-images.githubusercontent.com/8519407/132716040-7a6acde1-8402-4934-94de-4ac096cba90a.png)

**Comment by sunilyadav840** [comment]
As per narrator, Its read label with dropdown value. 
Please find the video link for reference.

https://drive.google.com/file/d/1FscYzyunGsiqgAi0y_j9skgHPGFeyn6F/view
